src/rpc/service_eth_test.go

```diff
@@ -79,10 +97,14 @@ func assertDynamicTxEquality(t *testing.T, cachedTx *types.Transaction, signedTx
 
 // First transaction gets sent and cache gets updated
 func TestSendRawTransaction_Success(t *testing.T) {
-	service := initTest(t)
+	service, mockDb := initTest(t)
 	rawTx1, signedTx, err := testdata.Tx(service.Processor.SigningKey, 1, big.NewInt(1))
 	assert.NoError(t, err, "Failed to create signed transaction")
 
+	mockDb.ExpectBegin()
+	mockDb.ExpectExec("INSERT INTO \"transaction_details\"").WithArgs(sqlmock.AnyArg(), sqlmock.AnyArg(), sqlmock.AnyArg(), sqlmock.AnyArg(), sqlmock.AnyArg(), sqlmock.AnyArg(), sqlmock.AnyArg()).WillReturnResult(sqlmock.NewResult(1, 0))
```

Comment:
If the mockDB is not used for any purpose here, could we remove it? We probably need to add it when we add the DB tests, but if it's just introducing unused code, should be removed.

Reply:
Yes, but it's now introducing unused code, and should be removed, we shouldn't be introducing unused code, even if we are intending to use it in the future.